variables.tf

```diff
@@ -67,3 +55,52 @@ variable "config_logs_prefix" {
   default     = "config"
   type        = "string"
 }
+
+# Service Switches
+variable "default_enable" {
+  description = "Create one bucket with all services as bucket keys."
+  default     = true
+  type        = "string"
+}
+
+variable "enable_cloudtrail" {
+  description = "Enable CloudTrail to log to the AWS logs bucket or the cloudtrail logs bucket."
+  default     = false
+  type        = "string"
+}
+
+variable "enable_cloudwatch" {
+  description = "Enable CloudWatch to log to the AWS logs bucket or the cloudwatch logs bucket."
```

Comment:
I'm not following this description. This should basically be the same as the others in that it allows CloudWatch to send logs to this S3 bucket.